src/vaadin-pdf-viewer.html

```diff
@@ -81,10 +83,45 @@
             /**
              * A represenentation of a document that has been read in. 
              */
-            _document: Object
+            _document: Object,
+            /**
+             * The title for the PDF shown in the header of component. It uses both the file name and
+             * the title in the PDF metadata if available.
+             */
+            _title: {
+              type: String,
+              value: "PDF"
+            },
+            /**
+             * Relative filename 
+             */
+            _filename: String,
+            /**
+             * The pdf metadata title
+             */
+            _pdfTitle: String
+
           };
         }
 
+        static get observers() {
+          return [
+            'setTitle(_pdfTitle, _filename)'
+          ];
+        }
+
+        setTitle(_pdfTitle, _filename) {
```

Comment:
No need in specifying parameters with underscore (can be just pdfTitle and filename)

Reply:
Done.